src/Bicep.Core/Semantics/ParamsSemanticModel.cs

```diff
@@ -114,6 +122,18 @@ public bool HasErrors()
             }
             return declarationsByAssignment.ToImmutableDictionary();
         }
+        /// <summary>
+        /// Gets all diagnostics raised by loading Bicep config for this template.
+        /// </summary>
+        private IEnumerable<IDiagnostic> GetConfigDiagnostics()
+        {
+            var diagnosticWriter = ToListDiagnosticWriter.Create();
```

Comment:
Are you using this?

Reply:
Left that in by mistake. Removed in latest commit.